### ISSUE TITLE
Staging Review finding: Heading structure doesn't match content

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104549]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** BAH-MBS-SelfServ
**Product:** Current - Pre-need Burial Requested change to Pre-Need Self Service
**Feature:** Pre-Need modernization and CaMEO Integration

## Findings details
**VA.gov Experience Standard - issue:** [User can determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

I encountered three situations where the heading hierarchy doesn't match the structure of the content:

- When an address can't be confirmed, the warning alert heading should be an H3
- When uploading supporting documents, the accordions should be H4s
- On the confirmation page after submitting the form, there's no H2 and the H3 and H4 hierarchy doesn't match the content

Screen reader users often use headings to navigate the page and understand the page structure. When heading levels don't match the content, users can get lost or confused.

## Recommended action

The best way to approach headings is to pretend you're back in 9th grade English class. Make a plain text outline of everything on the page, organized into big sections (H2s), subsections within those big sections (H3s), sub-subsections within those (H4s), etc.

Note that the form step indicator is itself hardcoded as an H2, and accordion trigger elements are themselves headings as well.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by carly-yamrus** [comment]
@shiragoodman Hi there. Followup question on the confirmation page heading structure.  For your comment: 
_On the confirmation page after submitting the form, there's no H2 and the H3 and H4 hierarchy doesn't match the content_


I think I understand this to mean the (You've submitted your application) will be an H2, and then the H4's **(Do you have more documents you need to submit?) and (What happens after I apply?)** should be H3s. Is this correct?  Thank you!

![Image](https://github.com/user-attachments/assets/9b8319dd-961c-42f6-aaae-6e387c7c6ef5)

**Comment by shiragoodman** [comment]
Hi @carly-yamrus - I didn't actually write this issue. It was from our accessibility specialist @briandeconinck. Brian, are you able to respond to the comment above?

**Comment by MichelleDieudonne** [comment]
We have created an enhancement story to complete this work: https://jira.devops.va.gov/browse/MBMS-92409.  This will be a small lift for us and will pulled into a PI and sprint as capacity allows.

**Comment by shiragoodman** [comment]
I'm sorry @carly-yamrus but Brian moved of Governance in May. It might be best to reach out to the Accessibility-DE team (ADE) for support. @lakper @coforma-jamie @sara-amanda is this something your team can assist with?

**Comment by lakper** [comment]
We can take a look!

**Comment by carly-yamrus** [comment]
@coforma-jamie very helpful thank you !
